### ISSUE TITLE
Method docblocks misplaced

### ISSUE THREAD

**Issue by czosel** [prettier/plugin-php#250]
Input:
```php
class Foo
{

    public function setTitle($title)
    {
    }

    /**
     * Another Comment
     */
    public function someMethod() {
    }
}
```

Output:
```php
class Foo
{
    public function setTitle($title)
    {
        /**
     * Another Comment
     */
    }
    public function someMethod()
    {

    }
}
```

**Comment by mgrip** [comment]
I think this got fixed in the latest updates to the parser - after https://github.com/prettier/plugin-php/pull/251 (which will update the parser version) gets deployed lets circle back and make sure

**Comment by mgrip** [comment]
Fixed in #251
